corim/profiles.go

```diff
@@ -74,11 +76,15 @@ func UnmarshalSignedCorimFromCBOR(buf []byte) (*SignedCorim, error) {
 // the data, they will be registered with the UnsignedCorim before it is
 // unmarshaled.
 func UnmarshalUnsignedCorimFromCBOR(buf []byte) (*UnsignedCorim, error) {
+	if !bytes.Equal(buf[:3], UnsignedCorimTag) {
```

Comment:
Should the comment on Line 74 not clarify that the CBOR payload is now expected to be tagged?

// // UnmarshalUnsignedCorimFromCBOR unmarshals a tagged UnsignedCorim from provided
// CBOR data.

Reply:
"tagged" is implied. It's not valid for an unsigned corim not to be tagged.